bin/citrea/tests/test_helpers/mod.rs

```diff
@@ -32,93 +32,76 @@ pub enum NodeMode {
     Prover(SocketAddr),
 }
 
-#[allow(clippy::too_many_arguments)]
 pub async fn start_rollup(
     rpc_reporting_channel: oneshot::Sender<SocketAddr>,
     rt_genesis_paths: GenesisPaths,
     rollup_prover_config: Option<ProverConfig>,
-    node_mode: NodeMode,
-    rollup_db_path: PathBuf,
-    da_db_path: PathBuf,
-    min_soft_confirmations_per_commitment: u64,
-    include_tx_body: bool,
-    rollup_config: Option<FullNodeConfig<MockDaConfig>>,
+    rollup_config: FullNodeConfig<MockDaConfig>,
     sequencer_config: Option<SequencerConfig>,
-    test_mode: Option<bool>,
-    deposit_mempool_fetch_limit: usize,
 ) {
     // create rollup config default creator function and use them here for the configs
-    let rollup_config = rollup_config.unwrap_or_else(|| {
-        create_default_rollup_config(include_tx_body, &rollup_db_path, &da_db_path, node_mode)
-    });
 
     let mock_demo_rollup = MockDemoRollup {};
 
-    match node_mode {
-        NodeMode::FullNode(_) => {
-            let span = info_span!("FullNode");
-            let rollup = CitreaRollupBlueprint::create_new_rollup(
-                &mock_demo_rollup,
-                &rt_genesis_paths,
-                rollup_config.clone(),
-            )
-            .instrument(span.clone())
+    if sequencer_config.is_some() && rollup_prover_config.is_some() {
+        panic!("Both sequencer and prover config cannot be set at the same time");
+    }
```

Comment:
The `assert_ne` did the same thing, no?

Reply:
No, it is allowed for them to be both None, which would fail `assert_ne`